README.md

```diff
@@ -1,7 +1,49 @@
-# Iroha JavaScript
+# Iroha2 JavaScript
 
 JavaScript SDK for [Iroha 2](https://github.com/hyperledger/iroha/tree/iroha2/).
 
+The Iroha 2 JavaScript library includes the following packages:
+
+- [`@iroha2/client`](./packages/client/) submits requests to the Iroha peer
+- [`@iroha2/monorepo-crypto`](./packages/crypto/):
```

Comment:
I'd suggest to not mention private utility packages in this list. `@iroha2/monorepo-crypto` is one of them.

Maybe replace it with

> - Iroha 2 Crypto:

or

> - `@iroha2/crypto-*`:

?

Reply:
```suggestion
- [`@iroha2/crypto-*`](./packages/crypto/):
```